google/earthenterprise
# Set default database

Adds a checkbox to the publish dialog for 3D databases that allows the user to make it the default database.

To verify functionality:

- Build and push a 3D database
- Publish the database, making sure to check the "Default Database" checkbox
- Verify that the "Default" indicator shows up in the database description on the admin page
- In Earth Client, enter only the server's base URL (without appending a database name), and verify that the expected database loads
- **Clear the EC cache**
- Perform the same steps with another database that has some obvious difference with the first. Make sure that the new database now loads for the base URL.

To verify documentation updates:

- Open index.html in `earthenterprise/docs/geedocs/5.3.3/html`
- Inspect the new page, "Publish default database". See if there are any issues with it.
- On the following pages, make sure the publish dialog screenshots include the "Default Database" checkbox

  - Pushing, Publishing, and Viewing a Database
  - Push and publish databases
  - Publish databases and portables
  - Search your POI vector data or map layer data
  - Display custom logos
  - Specifying Search Fields for Individual Layers
  - Configuring a Searchable Database
